index.bs

```diff
@@ -330,7 +330,6 @@ dictionary UADataValues {
   DOMString model;
 };
 
-[Exposed=Window]
 interface NavigatorUAData {
```

Comment:
Exposed is mandatory. You want `[Exposed=(Window,Worker)]`.

If Chrome's IDL compiler allows non-mandatory exposed, that's worth filing another bug (or one might already exist).

Reply:
Yup, adding it back (and it's mandatory for `self.NavigatorUAData` to not be undefined, so that seems fine)